### DISCUSSION TITLE
draw circle at xyz location with ijk vector

### DISCUSSION THREAD

**Discussion by JohnForeman** [pyvista/pyvista#4187] [Q&A]
I have a circle at location = [26.3878, 15.235, -22.7757] and normal = [0.75, 0.433013, 0.5].
when i draw the circle i can get the location correct but not the orientation.  is there a way with pyvista to simply use the ijk to set the circle noremal?

**Comment by JohnForeman** [comment]
i ended up making a very very short cylinder

**Comment by banesullivan** [comment]
For some reason the `Circle()` API was not implemented to leverage the translate API to allow the location and orientation to be set. We should fix this. It would look like this:

```py
import numpy as np
import pyvista as pv
from pyvista.utilities.geometric_objects import translate

circle = pv.Circle()
circle.rotate_x(90, inplace=True)
circle.rotate_z(90, inplace=True)
# In-place operation - assumes facing x-direction
# translate(circle, center=(0, 2, 1), direction=(0, 1, 0))
translate(circle, center=(26.3878, 15.235, -22.7757), direction=(0.75, 0.433013, 0.5))

pl = pv.Plotter()
pl.add_mesh(circle)
pl.show_grid()
pl.show()
```
![screenshot (6)](https://user-images.githubusercontent.com/22067021/230626108-3038e392-c31a-4735-b6c1-f92a90b3d2b7.png)
